### PR TITLE
Rework the installation of depopts

### PR DESCRIPTION
**Pull request by samoht**
Do not run the tests while testing DEPOPTS.

### CODE REVIEW

File: travis_opam.ml [hunk 1]
```diff
@@ -117,38 +119,39 @@ List.iter pin pins;
...
+begin (* optioanal dependencies *)
+  let depopts_run = match depopts_run with
+    | [] | ["false"] -> None
+    | ["*"] -> (* query OPAM *)
       ?|> "opam show %s | grep -oP 'depopts: \\K(.*)' | sed 's/ | / /g'" pkg
-    in
-    with_opambuildtest (fun () -> install_with_depopts args depopts)
-  | depopts ->
-    with_opambuildtest (fun () -> install_with_depopts args (depopts *~ " "))
+      |>  some
```
Comment (dsheets) [inline travis_opam.ml]: `|>` is not available on all supported OCaml versions.

**Comment by avsm** [comment]
Please revert this quickly @samoht -- it breaks a lot of external repositories.

**Comment by samoht** [comment]
See https://github.com/ocaml/ocaml-travisci-skeleton/pull/59.

**Comment by dsheets** [comment]
We should turn on Travis for this repository...

**Comment by samoht** [comment]
good idea, I'll add it.

**Comment by samoht** [comment]
Done in https://github.com/ocaml/ocaml-travisci-skeleton/pull/60

**Pull request merged**